python/cudf/cudf/core/dataframe.py

```diff
@@ -3338,6 +3344,9 @@ def _insert(self, loc, name, value, nan_as_null=None, ignore_index=True):
         value = column.as_column(value, nan_as_null=nan_as_null)
 
         self._data.insert(name, value, loc=loc)
+        if was_empty:
+            # TODO: Create a cudf_to_pandas_dtype function to use here
```

Comment:
We have `cudf_dtype_to_pa_type`, and I assume it's possible to convert a pyarrow type to a pandas type already, so would that work?